tests/integrations/minter/config.yml

```diff
@@ -0,0 +1,27 @@
+user:
```

Comment:
That's a good idea, actually I was afraid to over constrain test configs doing this (since there are no fundamental reason to link our test settings), but with the following proposition we could get the better of all worlds I think:

```
tests/
├── config.yaml
├── integrations/
│   ├── config.yaml
│   ├── library.cairo
│   ├── minter/
│   │   ├── config.yaml
│   │   └── test_...
│   ├── yielder/
│   │   ├── config.yaml
│   │   └── test_...
├── units/
│   ├── config.yaml
│   ├── library.cairo
│   ├── minter/
│   │   ├── config.yaml
│   │   └── test_...
│   ├── yielder/
│   │   ├── config.yaml
│   │   └── test_...
```

With the following priority (minter as instance):
- 1. tests/integrations/minter/config.yaml
- 2. tests/integrations/config.yaml
- 3. tests/config.yaml

Doing this, if we need to override a top level config, we can just rewrite it in a deeper level.

Note: Additionaly,  I'll probably refactor `signers` and `user` into `signers`.

Reply:
@cloudvenger ,
A counter argument could also be, do we want less readibiliy by spreading a test config into several config files ? (In favor of mutualize few lines)

Reply:
Your proposal is perfect for me ;) 

For me, it's not a problem to have multi config.yml but what is important, is not duplicate the same variables with the same values. So, be able to overwrite it if necessary but not duplicate teh value.

So, you proposal is good for me :)